app/src/main/java/com/duckduckgo/app/di/AppComponent.kt

```diff
@@ -79,7 +80,8 @@ import javax.inject.Singleton
         CertificateTrustedStoreModule::class,
         WelcomePageModule::class,
         HttpsPersisterModule::class,
-        EmailModule::class
+        EmailModule::class,
+        BookmarksModule::class,
```

Comment:
Ditto, I'd suggest to remove this in favor of using `@ContributesTo(AppObjectGraph::class)` right in the definition of the dagger module